test/load/test_load_tx_ethereum.py

```diff
@@ -0,0 +1,273 @@
+import time
+import threading
+import copy 
+
+import siftool_path
+from siftool import eth, test_utils, sifchain
+from siftool.eth import NULL_ADDRESS
+from siftool.common import *
+from web3.eth import Contract
+import web3
+
+from eth_typing import (
+    ChecksumAddress,
+)
+
+# In separate window: test/integration/framework/siftool run-env
+# test/integration/framework/venv/bin/python3 test/load/test_load_tx_ethereum.py
```

Comment:
This too, it is a Python program, so it could conceivably be launched by the test suite and include code to clean up when tests finish.